CODIGO/frmLobbyBattleground.frm

```diff
@@ -0,0 +1,249 @@
+   End
+End
+Attribute VB_Name = "frmLobbyBattleground"
+Attribute VB_GlobalNameSpace = False
+Attribute VB_Creatable = False
+Attribute VB_PredeclaredId = True
+Attribute VB_Exposed = False
+Dim MouseIndex As Integer
+Dim Scroll As Integer
+Const MAX_LIST As Integer = 6
+
+Private Sub btnCrear_Click()
+    frmCreateBattleground.Show
+    Unload Me
+End Sub
+
+Private Sub Form_Load()
+    ListRefresh
+End Sub
+Private Sub pEvents_Click()
+On Error GoTo ErrHandler:
+    Dim Password As String
+    If MouseIndex > 0 And MouseIndex <= UBound(LobbyList) Then
+        If LobbyList(MouseIndex).IsPrivate Then
+            Password = InputBox("Este evento tiene contraseña, por favor ingresala:")
+            If Len(Password) = 0 Then Exit Sub
+        End If
+        Call WriteParticipar(LobbyList(MouseIndex).id, Password)
+        Unload Me
+    ElseIf MouseIndex = -1 Then
+        Scroll = Scroll - 1
+        If Scroll < 0 Then Scroll = 0
+    ElseIf MouseIndex = -2 Then
+        Scroll = Scroll + 1
+        If Scroll + MAX_LIST > UBound(LobbyList) Then Scroll = UBound(LobbyList) - MAX_LIST
+        If Scroll < 0 Then Scroll = 0
+    End If
+    Exit Sub
+ErrHandler:
+    Call RegistrarError(Err.Number, Err.Description, "frmLobbyBattleground.Click", Erl)
+    Resume Next
+End Sub
+
+Private Sub pEvents_MouseMove(Button As Integer, Shift As Integer, X As Single, Y As Single)
+    Dim i As Integer
+    MouseIndex = 0
+    For i = 1 To MAX_LIST
+        If X >= 810 And X <= 810 + 70 And Y >= i * 45 And Y <= 35 + i * 45 Then
+            MouseIndex = i + Scroll
+            Exit For
+        End If
+    Next i
+    If X >= pEvents.Width - 15 And X <= pEvents.Width - 5 And Y >= 45 And Y <= 50 Then
+        MouseIndex = -1
+    End If
+    If X >= pEvents.Width - 15 And X <= pEvents.Width - 5 And Y >= pEvents.Height - 15 And Y <= pEvents.Height - 10 Then
+        MouseIndex = -2
+    End If
+    ListRefresh
+End Sub
+
+Private Sub ListRefresh()
+On Error GoTo ErrHandler:
+    Dim OffX As Integer
+    Dim Offy As Integer
+    With pEvents
+        pEvents.Cls
+        .ForeColor = vbWhite
+        .FontSize = 10
+        OffX = 10
+        Offy = 10
+        
+        .CurrentX = OffX
+        .CurrentY = Offy
+        pEvents.Print "Descripción"
+        
+        OffX = OffX + 140
+        .CurrentX = OffX
+        .CurrentY = Offy
+        pEvents.Print "Tipo"
+        
+        OffX = OffX + 100
+        .CurrentX = OffX
+        .CurrentY = Offy
+        pEvents.Print "Tamaño de grupos"
+        
+        OffX = OffX + 130
+        .CurrentX = OffX
+        .CurrentY = Offy
+        pEvents.Print "Formato de equipos"
+        
+        OffX = OffX + 130
+        .CurrentX = OffX
+        .CurrentY = Offy
+        pEvents.Print "Límite de nivel"
+        
+        OffX = OffX + 130
+        .CurrentX = OffX
+        .CurrentY = Offy
+        pEvents.Print "Jugadores"
+        
+        OffX = OffX + 100
+        .CurrentX = OffX
```

Comment:
do we scroll on x? why?

Reply:
it's not a scroll, its the x position of where you are printing the column